### DISCUSSION TITLE
整合包vsLite和vsMega的使用方法

### DISCUSSION THREAD

**Discussion by rhein999** [hooke007/mpv_PlayKit#364] [mpv 通用]
翻遍了所有文档和讨论都没有找到整合包vsLite和vsMega的使用方法，难道是直接把要用到vs滤镜移动到标准版对vs文件夹内？或者直接移动覆盖？或者在conf文件内加入整合包内vs滤镜的路径？

**Comment by hooke007** [comment]
难道看不出来路径是完全对应mpv-lazy做的吗，解压覆盖一个压缩包的难点到底在什么地方？

**Reply by rhein999** [reply]
我之前就是这样做的，但有很多要覆盖的重复文件造成了一些迷惑
